sdk/communication/azure-communication-rooms/src/main/java/module-info.java

```diff
@@ -2,17 +2,18 @@
 // Licensed under the MIT License.
 
 module com.azure.communication.rooms {
+    requires com.azure.json;
```

Comment:
Not needed as `azure-core` requires this transitively, so downstream libraries don't need this.